common/repo/GitRepo.java

```diff
@@ -461,6 +461,11 @@ public boolean hasChanges(String branch) {
     return (getCommits(branch).size() > 1) || (!getUncommittedFiles().isEmpty());
   }
 
+  @Override
+  public String getPatch(String commitOrBranch, String filename) {
```

Comment:
s/commitOrBranch/referenceCommitOrBranch
According to the docs, this is for "changes you have in your working tree relative to the named <commit>".
https://git-scm.com/docs/git-diff